warehouse/templates/email/removed-package/subject.txt

```diff
@@ -0,0 +1,19 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block content %}
+  {% trans project=project %}The package {{ project }} has been deleted.{% endtrans %}
```

Comment:
Again, for consistency I think we should have this be "The project {{ project }} has been deleted."